### ISSUE TITLE
Incorrect parsing for contextual keyword `is`

### ISSUE THREAD

**Issue by mrm-rideos** [typescript-eslint/typescript-eslint#749]
This is the same issue as #678, credit to @svr93 for the description. It seems the `is` keyword needs to be added as part of that fix as well. I'll attempt a PR as well.

**What code were you trying to parse?**

```ts
import { Map, is } from 'immutable'
```

**What did you expect to happen?**
token type `Identifier` with value `is`

**What actually happened?**
token type `Keyword` with value `is`

**Versions**

| package                                | version |
| -------------------------------------- | ------- |
| `@typescript-eslint/typescript-estree` | `1.13.0` |
| `TypeScript`                           | `3.5.3` |
| `node`                                 | `10.15.3` |
| `npm`                                  | `6.4.1` |
